hangman.js

```diff
@@ -0,0 +1,117 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Overall well thought out app with good animal theme. There are few glitches that are related to faulty logics and DOM element management. Please do revisit those glitches later on your free time. All of them require some tweaks to the code or adding right DOM manipulation methods to solve. Good job and looking forward to see your next projects!